proxy/log4j2-plugin/build.gradle.kts

```diff
@@ -1,3 +1,10 @@
+tasks.withType<JavaCompile> {
+    options.compilerArgs.addAll(listOf(
```

Comment:
What’s the reason for this?

Reply:
Otherwise it won't compile because the next log4j had introduced this change. See https://github.com/apache/logging-log4j2/pull/3125#issuecomment-2517245837